index.html

```diff
@@ -17,14 +17,15 @@
           </div>
 
           <nav class="desktop-nav">
+            <div class="x-nav"><img src="images/closeMark.png" alt="" /></div>
             <ul>
-              <li><a href="#">Portfolio</a></li>
-              <li><a href="#">About</a></li>
-              <li><a href="#">Contact</a></li>
+              <li><a href="#ancre1">Portfolio</a></li>
+              <li><a href="#ancre2">About</a></li>
+              <li><a href="#ancre3">Contact</a></li>
```

Comment:
- [ ] Great work and UI design I like it, but we have some issues with spacing and alignments in the header (mobile view), if you notice below in image comparison between the Figma design and your implementation, our hamburger menu has more space to the right than it should be, and it's not aligned properly with `My logo`

Your implementation | Figma Design
:------:|:-----:
![image](https://user-images.githubusercontent.com/87689547/157380145-a5bd7d72-35fc-4019-922c-c372de93696f.png) | ![image](https://user-images.githubusercontent.com/87689547/157380277-47c76f01-048f-4a88-a40a-5f7c5d5bdc12.png)

Reply:
Thank you, I just did that